airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/ConfigFetchActivityImpl.java

```diff
@@ -32,25 +32,35 @@ public class ConfigFetchActivityImpl implements ConfigFetchActivity {
 
   @Override
   public ScheduleRetrieverOutput getTimeToWait(final ScheduleRetrieverInput input) {
+    final boolean noSchedule;
```

Comment:
I keep removing `final` from this line and IntelliJ keeps adding it.  I don't understand why.  
I understand final to be more or less a constant variable, and I fully intend to change it

Am I adopting the pattern incorrectly or is IntelliJ configured incorrectly?

Reply:
good question - our default style guide adds final to variables that aren't changing. Because this isn't being assigned to Intellij is probably detecting this as unchanging and thus adds final to it.